app/javascript/lib/rich_text/unfurl/lib/opengraph_embed_operation.js

```diff
@@ -61,20 +86,37 @@ export default class OpengraphEmbedOperation {
   }
 
   #generateOpengraphEmbedHTML(embed) {
+    const escapedTitle = this.#escapeHtml(truncateString(embed.title, 560))
+    const escapedDescription = this.#escapeHtml(truncateString(embed.description, 560))
+    const escapedImage = this.#escapeHtml(embed.image)
+    
     return `<actiontext-opengraph-embed class="${this.#isTwitterAvatar(embed) ? UNFURLED_TWITTER_AVATAR_CSS_CLASS : ''}">
       <div class="og-embed">
         <div class="og-embed__content">
-          <div class="og-embed__title">${truncateString(embed.title, 560)}</div>
-          <div class="og-embed__description">${truncateString(embed.description, 560)}</div>
+          <div class="og-embed__title">${escapedTitle}</div>
+          <div class="og-embed__description">${escapedDescription}</div>
         </div>
         <div class="og-embed__image">
-          <img src="${embed.image}" class="image" alt="" />
+          <img src="${escapedImage}" class="image" alt="" />
         </div>
       </div>
     </actiontext-opengraph-embed>`
   }
 
+  #escapeHtml(text) {
+    const div = document.createElement('div')
+    div.textContent = text
+    return div.innerHTML
+  }
```

Comment:
I think we can remove this sanitization completely since [the unfurl payload is already sanitizied](https://github.com/basecamp/once-campfire/blob/a80c6d7e91a6c5192e5bf38db31888229a5d402d/app/models/opengraph/metadata.rb#L17).